packages/form-field/src/FormFieldFeedback/FormFieldFeedback.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Size, spacing, typeScales } from '@leafygreen-ui/tokens';
+
+export const containerStyles = css`
+  display: flex;
+  gap: ${spacing[100]}px;
+`;
+
+export const spacingTop = css`
+  padding-top: ${spacing[100]}px;
+`;
+
+export const hideContainerStyle = css`
+  opacity: 0;
+`;
+
+export const iconWrapperStyles = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const getIconWrapperHeight = (size: Size) => {
+  return css`
+    height: ${size === Size.Large ? 24 : typeScales.body1.lineHeight}px;
```

Comment:
Is `typeScales.body2.lineHeight` not `24`?

Reply:
thanks for syncing offline on this. I've corrected the `typeScales` token and swapped this with the corrected `typeScales` token